src/main/java/duke/exception/DukeException.java

```diff
@@ -0,0 +1,6 @@
+package duke.exception;
+
+public class DukeException extends Exception {
+
+    // Nothing here
```

Comment:
umm no need to add comment here (consider this for other classes in exception as well